api/cpp/include/slint-platform.h

```diff
@@ -276,17 +275,26 @@ class Platform
     /// Returns a new WindowAdapter
     virtual std::unique_ptr<WindowAdapter> create_window_adapter() = 0;
 
-#ifdef SLINT_FEATURE_FREESTANDING
+#if defined(SLINT_FEATURE_FREESTANDING) || defined(DOXYGEN)
     /// Returns the amount of milliseconds since start of the application.
     ///
     /// This function should only be implemented  if the runtime is compiled with
     /// SLINT_FEATURE_FREESTANDING
-    virtual std::chrono::milliseconds duration_since_start() const
-    {
-        return {};
-    }
+    virtual std::chrono::milliseconds duration_since_start() const { return {}; }
 #endif
 
+    /// The clipboard, used in Platform::clipboard_text and PLatform::set_clipboard_text
+    enum class Clipboard {
+        /// This is the default clipboard used for text action for Ctrl+V,  Ctrl+C.
+        /// Corresponds to the secondary clipboard on X11.
```

Comment:
As per [ICCCM](https://x.org/releases/X11R7.6/doc/xorg-docs/specs/ICCCM/icccm.html#selection_atoms).

```suggestion
        /// Corresponds to the secondary selection on X11.
```